src/Components/Web/src/Forms/AntiforgeryStateProvider.cs

```diff
@@ -0,0 +1,48 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Components.Forms;
+
+/// <summary>
+/// Provides access to the antiforgery token associated with the current session.
+/// </summary>
+public class AntiforgeryStateProvider : IDisposable
+{
+    private const string PersistenceKey = nameof(AntiforgeryRequestToken);
```

Comment:
Could we change this key to something that indicates it's a private implementation detail? Otherwise people might start reading this token name directly from the persistence store and then we'll be locked to it.

```suggestion
    private const string PersistenceKey = $"__internal__{nameof(AntiforgeryRequestToken)}";
```